go/programs/bubble_sort.go

```diff
@@ -0,0 +1,51 @@
+package programs
+
+import "fmt"
+
+func BubbleSort(array []int) []int {
+	size := len(array)-1
+	var iteration, swap int
+	for _= range array{
+		swap = 0
+		for index:= 1; index <= size-iteration; index++{
+			if array[index-1] > array[index]{
+				swap++
+				temp := array[index]
+				array[index] = array[index-1]
+				array[index-1] = temp
```

Comment:
create a helper method to swap variables.

Reply:
okayy